### PR TITLE
Create Logo component

### PR DESCRIPTION
**Pull request by alex6061**

### CODE REVIEW

File: js/components/features/Logo/Logo.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
```
Comment (nagorapro) [inline js/components/features/Logo/Logo.js:1]: import { IconLogo } from './ui/IconLogo.js';

File: js/components/features/Logo/Logo.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <path fill-rule="evenodd" clip-rule="evenodd" d="M27.8694 1.51819C29.0274 1.51819 29.9659 2.44599 29.9659 3.59012C29.9659 4.73425 29.0274 5.66205 27.8694 5.66205C26.7118 5.66205 25.7732 4.73425 25.7732 3.59012C25.7732 2.44599 26.7118 1.51819 27.8694 1.51819Z" fill="black"/>
+      </svg>
```
Comment (nagorapro) [inline js/components/features/Logo/Logo.js:18]: Delete

Comment (nagorapro) [inline js/components/features/Logo/Logo.js:18]: Delete
Insert ${IconLogo()}

Comment (nagorapro) [inline js/components/features/Logo/Logo.js:18]: Delete
Insert ${IconLogo()}

File: js/components/features/Logo/Logo.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ */
+
+export const Logo = (parrentClassName) => {
+
+  const currentClassName = parrentClassName
+    ? `${parrentClassName}__logo`
+    : `logo`;
+
+  return `
+    <div class="${currentClassName}">
```
Comment (nagorapro) [inline js/components/features/Logo/Logo.js:14]: button

**Review by nagorapro — COMMENTED** [review]
Need fix

File: js/components/features/Logo/Logo.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { IconLogo } from '../../ui/IconLogo/IconLogo.js';
+import { data } from './../../../data.js';
```
Comment (nagorapro) [inline js/components/features/Logo/Logo.js:2]: Bad

File: js/components/features/Logo/Logo.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+/**
+ * @function Logo 
+ * @param {data} logo
```
Comment (nagorapro) [inline js/components/features/Logo/Logo.js:6]: Bad

**Review by nagorapro — COMMENTED** [review]
Need fix

File: js/components/features/Logo/Logo.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import {IconLogo} from '../../ui/IconLogo/IconLogo.js';
```
Comment (nagorapro) [inline js/components/features/Logo/Logo.js:1]: Use spaces

**Review by nagorapro — APPROVED** [review]

File: js/data.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 export const data = {
   en: { 
     header: {
-      logo: {},
+      logo:{},
```
Comment (nagorapro) [inline js/data.js:4]: ...

**Review by nagorapro — APPROVED** [review]

File: js/data.js (hunk 7 above)
Comment (alex6061) [reply js/data.js:4]: Need to add a space?

**Pull request merged**